content/applications/inventory_and_mrp/barcode/operations/barcode_nomenclature.rst

```diff
@@ -1,84 +1,211 @@
-========
-Overview
-========
+============================
+Default barcode nomenclature
+============================
 
-There are different situations in which barcode nomenclatures can be
-useful. A well-known use case is the one of a point of sale which sells
-products in bulk, in which the customers will scale their products
-themselves and get the printed barcode to stick on the product. This
-barcode will contain the weight of the product and help compute the
-price accordingly.
+.. |UPC| replace:: :abbr:`UPC (Universal Product Code)`
+.. |EAN| replace:: :abbr:`EAN (European Article Number)`
+.. |GTIN| replace:: :abbr:`GTIN (Global Trade Item Number)`
+.. |GTINs| replace:: :abbr:`GTINs (Global Trade Item Numbers)`
 
-Create a Barcode Nomenclature
-=============================
+*Barcode nomenclatures* define how barcodes are recognized and categorized. When a barcode is
+scanned, it is associated to the **first** rule with a matching pattern. The pattern syntax is
+described in Odoo's nomenclature list using a regular expression, and a barcode is successfully read
+by Odoo if its prefix and/or length matches the one defined in the barcode's rule.
 
-Odoo supports Barcode Nomenclatures, which determine the mapping and
-interpretation of the encoded information. You can configure your barcode nomenclature being in
-:ref:`developer mode <developer-mode>`.
-To do so, go to :menuselection:`Inventory --> Configuration --> Barcode Nomenclature`.
+Odoo *Barcode* supports UPC (Universal Product Code), EAN (European Article Number), and GS1
+encoding. The pre-configured nomenclatures in Odoo are *Default Nomenclature* and *Default GS1
+Nomenclature*. The default nomenclature uses |UPC| and |EAN| encoding, and supports |UPC|/|EAN|
+conversion.
 
-You can create a barcode nomenclature from there, and then add a line to
-create your first rule.
+.. important::
+   |UPC| and |EAN| barcodes **must** be `purchased from GS1
+   <https://www.gs1.org/standards/get-barcodes>`_ to use these barcodes. GS1 is the **only**
+   official provider of |UPC|/|EAN| and GS1 |GTINs| globally.
```

Comment:
```suggestion
   official provider of |UPC|/|EAN| and GS1 |GTINs| in the world.
```
just sounds a bit cleaner with this phrasing

Reply:
Suggestion implemented ✅